src/assets/stylesheets/index.scss

```diff
@@ -1148,7 +1149,7 @@ section:focus >,
   border-radius: 50%;
   line-height: 60px;
   text-align: center;
-  font-weight: 500;
+  font-weight: var(--bs-body-font-weight);
```

Comment:
This attribute can be removed, since it will automatically take the body font weight.

Reply:
Yes.